devise_security_extension.gemspec

```diff
@@ -20,7 +20,11 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.required_ruby_version = '>= 2.1.0'
 
-  s.add_runtime_dependency 'railties', '>= 3.2.6', '< 6.0'
+  if RUBY_VERSION >= '2.4'
```

Comment:
Is pre Rails 4.2.8 compatibility important? If so, is this the way you want to support it? Ruby 2.4+ requires Rails 4.2.8+ because of changes in the `json` gem.

Reply:
IMO, its easy enough to upgrade to 4.2.[8, 9, 10] we should not need to support previous versions